### PR TITLE
Implement grapevine user search by name or email.

### PR DESCRIPTION
**Pull request by arbianchi**
Year and position search not yet implemented. Was curious how 'year' was going to work since that's not a column in the Users table...

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -58,7 +66,6 @@ def user_params
     :password,
     :phone,
     :pinterest_name,
-    :privacy_settings,
```
Comment (maebeale) [inline app/controllers/users_controller.rb]: @arbianchi why did you remove privacy settings here?

**Comment by maebeale** [comment]
@arbianchi if you make another class method similar to search, but prefix it with `joins(positions_helds: :program)`, you can then convert date to text and search using LIKE. would love to pair program on this if you'd like!

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (arbianchi) [reply app/controllers/users_controller.rb]: K, let's add positions to search. Didn't mean to remove privacy settings..that was work from a different PR. Will fix.

**Review by maebeale — APPROVED** [review]
